### DISCUSSION TITLE
Build app outside of AppBridge

### DISCUSSION THREAD

**Discussion by RalpinMcDjango** [Kyon147/laravel-shopify#19] [General]
Hi folks,

Been playing around with this library for a while and the app I'm building would benefit from being hosted outside of AppBridge, and on it's own dashboard.

I started a fresh repo to test ```SHOPIFY_APPBRIDGE_ENABLED=false``` and i disabled the emebed application from the Shopify partner dashboard but if i click to open the app from the partner dashboard i get a loading page with no content.
![image](https://user-images.githubusercontent.com/22886457/208142699-c76e2894-2e58-49bf-a43d-504ab6946217.png)

I can see the redirect path seems to be looking for the homepage ```/``` which is named ```home``` extending the  ```shopify-app::layouts.default``` layout template.

This works as expected within AppBridge as an iframe, what have i missed?

**Comment by Kyon147** [comment]
Hi @RalpinMcDjango 

The app only supports embedded apps at the moment. The original was the same, the functionality for out of iframe apps was lost somewhere between v10 and v16 because of all the cookie changes, ITP, JWY updates.

There was a PR on the original which was open and looking for someone to continue the work - so you could take that work and add to it if you wish.

I have no plans to work on the non-embed version at the moment as I personally have no use for it but would love for someone to contribute a PR to bring the feature back in.
